indico/modules/rb/client/js/modules/roomList/RoomList.jsx

```diff
@@ -176,6 +206,12 @@ class RoomList extends React.Component {
                     <div styleName="actions">
                       {selectionMode ? (
                         <>
+                          <ResponsivePopup
+                            trigger={
+                              <Button icon={selectAllIcon} onClick={this.selectAll} circular />
+                            }
+                            content={selectAllText}
+                          />{' '}
```

Comment:
```suggestion
                          <ResponsivePopup
                            trigger={
                              <Button 
                                 icon={selectAllIcon}
                                 onClick={this.selectAll}
                                 primary={this.allSelected()}
                                 circular
                              />
                            }
                            content={selectAllText}
                            position="bottom center"
                          />{' '}
```
In addition to that `check square` icon for both, we can also add a primary toggle for the button (this helps to improve the UX a bit - as well as tweaking the positioning the pop-up).